project.clj

```diff
@@ -1,58 +1,56 @@
-(def ks-version "1.3.0")
-(def tk-version "1.4.0")
-
 (defproject puppetlabs/trapperkeeper-metrics "0.4.3-SNAPSHOT"
   :description "Trapperkeeper Metrics Service"
   :url "http://github.com/puppetlabs/trapperkeeper-metrics"
 
+  :min-lein-version "2.7.1"
+
   :pedantic? :abort
 
-  :dependencies [[org.clojure/clojure "1.7.0"]
+  :parent-project {:coords [puppetlabs/clj-parent "0.2.4"]
+                   :inherit [:managed-dependencies]}
+
+  :dependencies [[org.clojure/clojure]
 
                  ;; begin version conflict resolution dependencies
-                 [clj-time "0.11.0"]
-                 [commons-codec "1.9"]
-                 [org.clojure/tools.macro "0.1.5"]
-                 [org.clojure/tools.reader "1.0.0-alpha1"]
-                 [prismatic/schema "1.1.0"]
-                 [slingshot "0.12.2"]
-                 [commons-io "2.4"]
-                 [ring/ring-servlet "1.4.0"]
+                 [clj-time]
+                 [commons-codec]
+                 [org.clojure/tools.macro]
+                 [org.clojure/tools.reader]
+                 [prismatic/schema]
+                 [slingshot]
+                 [commons-io]
+                 [ring/ring-servlet]
                  ;; end version conflict resolution dependencies
```

Comment:
Sound's good. Any others we should get rid of that weren't added in the "conflict resolution dependencies" section? For example, it seems like `ring/ring-core` could be pulled in by `puppetlabs/ring-middleware`